Gemfile

```diff
@@ -3,21 +3,19 @@ source "http://rubygems.org"
 # try to slowly migrate to v2 of the aws api
 gem 'aws-sdk', '~> 2'
 
-gem 'dynect_rest'
+gem 'dynect_rest', '= 0.4.6'
```

Comment:
only this version?

Reply:
I have a vague memory that versions starting with 0 don't necessarily follow the standard versioning conventions, so this was an attempt to be cautious. FWIW it appears that version is 5 years old and the latest version: https://rubygems.org/gems/dynect_rest/versions/0.4.6